src/util/services/firebase.service.ts

```diff
@@ -1,4 +1,5 @@
-import firebase from 'firebase'
+import firebase from 'firebase';
+
 const config = {
   apiKey: "AIzaSyBTdmYHtu9q1m-heSRK91cuy4b7ZK73cvs",
```

Comment:
Let's move these keys into a .env file so that people can't mess with the database.